pkg/capture/file/timestamp.go

```diff
@@ -0,0 +1,25 @@
+package file
+
+import (
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+type Timestamp struct {
+	time.Time
+}
+
+const captureFileNameTimestampFormat string = "2006#01#02#15#04#05UTC"
+
+func (timestamp *Timestamp) TimestampToString() string {
```

Comment:
Sorry, I missed a detail in my earlier explanation. In Go, if you want to say "this is how this type should be represented as a string," you should implement [`fmt.Stringer`](https://pkg.go.dev/fmt#Stringer). That allows you to just pass this to basically anything in `fmt` like this:

```go
now := &Timestamp{time.Now().UTC()}
fmt.Println("The time is", now)
```

Instead of having to do this (and discover that the method exists):

```go
now := &Timestamp{time.Now().UTC()}
fmt.Println("the time is", now.TimestampToString())
```

So all you should need to do is this:
```suggestion
func (timestamp *Timestamp) String() string {
```

Reply:
Very cool, thanks! Didn't know about this.